### ISSUE TITLE
Establish KRR synch-job/trigger

### ISSUE THREAD

**Issue by olebhansen** [Altinn/altinn-profile#188]
### Description

Blocked by #187 

This task is to trigger the feature from #187 on a schedule. 

The plan is to trigger trough an scheduled curl-call to an internal/unexposed API call, as is the pattern for the other similar jobs in other apps. Storage is using the same feature to start cleanup jobs and Notifications is using it to trigger different order processing tasks. The jobs are defined as kubernetes jobs in the helm charts (altinn-studio-ops repo in Azure DevOps).

The endpoints should not be visible for the ApiExplorer (OpenAPI spesification).

### Additional info
NB: the discussion was also to run the code in the job itself. This is not in scope now (for the sake of uniformity).

The batch in Altinn 2 runs once every 10 minutes.

### Tasks
- [x] Define a k8 job that calls the endpoint to perform the KRR sync one every x minutes. (Same interval as in Altinn 2).

### Acceptance criteria
- [ ] The job is started automatically at the configured interval.

**Comment by SandGrainOne** [comment]
Jobs have been started in all environments and I can see new updates having been processed base on the updated metadata. Metadata is being updated every 10 minutes.
